docs/proposals/31-inplace-node-update.md

```diff
@@ -236,43 +261,59 @@ metadata:
   name: pool-01-original
   namespace: default
 spec:
-  type: <os-type>
-  version: <operating-system-version>
+  type: gardenlinux
+  osVersion: 1631.0
+  kubeletVersion: 1.30.4
+  credentialRotation:
+    caRotationLastInitiationTime:
+    serviceAccountKeyRotationLastInitiationTime:
   units:
     - ...
   files:
     - ...
+status:
+  inPlaceUpdateConfig:
+    osUpdateScriptPath: /opt/gardener/bin/inplace-update.sh
+    osUpdateScriptArgs:
+      - --version
+      - 1631.0
+      - --repo
+      - <someOCIregistry>
```

Comment:
What is the benefit of having this in two separate fields? GNA will just always "blindly" call it, or?

Reply:
> GNA will just always "blindly" call it, or?

Yeah, OS extensions are free to use any flags of their choice.

Reply:
OK, so what is the point of having two fields for this (instead of one where the OS extension just puts the entire command with all flags)?

Reply:
Fair enough, updated the doc.